doc/tutorials/modeling-incidents-in-ctim.md

```diff
@@ -0,0 +1,226 @@
+
+The CTIM incident model is intended to help organizations track data that pertain to the *incident itself*.  These include (but are not limited to) factors such as:
+- The current status of the incident
+- Important times in the incident's lifecycle, such as when it was first discovered or reported, opened, closed, remediated, or rejected.
+- How the incident was discovered
+- Personnel who are assigned to the incident
+
+The details of the various threats and network events that pertain to the incident, as well as the details of potential mitigation or remediation steps are captured in other parts of the CTIM data model, and associated to the incident via `relationship` entities.
+
+#### 1.8.1: Incident Field Summary
+
+In addition to the [Common CTIM Entity Properties](https://github.com/threatgrid/ctim/blob/master/doc/tutorials/modeling-threat-intel-ctim.md#part-1-ctim-model#11-common-ctim-entity-properties) described in the parent tutorial, CTIM `incident` entities contain the following fields:
+
+##### 1.8.1.1: Required Incident Fields
+- **type**: This, **must** be the string `"incident"`.
+- **confidence**: Must be one of `["Info", "Low", "Medium", "High", "None", "Unknown"]`.
+- **status**: Must be one of `["New", "Open", "Stalled", "Containment Achieved", "Restoration Achieved", "Incident Reported", "Closed", "Rejected"]`
+- **incident_time**: Must be an `IncidentTime` inline data type, which have the following fields:
+
+###### 1.8.1.1.1: `IncidentTime` inline data type
+`IncidentTime` are inline data types similar to observables in that they are not top level entities, but are structured entities that exist inline within a top level entity.
+
+**Required `IncidentTime` fields**
+- **opened**: The time this incident was opened.
+
+**Optional `IncidentTime` fields**
+- **discovered**: The time this incident was first discovered, if knowledge of the incident originated from within the organization.
+- **reported**: The time this incident was first reported to the organization, if knowledge of the incident came from a source or service outside of the organization.
+- **remediated**: The time this incident was remediated.
+- **closed**: The time this incident was closed.
+- **rejected**: The time this incident was rejected.
+
+##### 1.8.1.2: Optional Incident Fields
+- **categories**: One or more categories for this incident. If included, must be a vector containing one or more of the string values defined in the [CTIM `incident-category` vocabulary](https://github.com/threatgrid/ctim/blob/487a5165276fe94957341a9516603dfe5c3ab2a5/src/ctim/schemas/vocabularies.cljc#L86).  This is an open vocabulary, so you can define your own values, but we strongly encourage you to suggest additional vocabulary entries if there is not a suitable one already defined.
```

Comment:
I would not link to the code and list it here, for the simple reason that in few commits the line will not be the same.

Reply:
It's a permalink (note the hash), and the vocab schema rarely changes.